Applications/ClientWebPortal/Source/webapp/src/utils/array-string-key-generator/index.js

```diff
@@ -0,0 +1,21 @@
+export default class {
+  get(values) {
+    for (let i=0; i<values.length; i++) {
```

Comment:
Don't loop twice over the array.  Just loop once and check for null as you iterate over the array.

Reply:
Got it.